code/modules/mob/living/carbon/human/species.dm

```diff
@@ -561,6 +561,9 @@ GLOBAL_LIST_EMPTY(features_by_species)
 	var/obj/item/bodypart/head/noggin = species_human.get_bodypart(BODY_ZONE_HEAD)
 
 	if(noggin && !(HAS_TRAIT(species_human, TRAIT_HUSK)))
+		for(var/head_overlay in noggin.generate_body_overlay(species_human))
+			standing += head_overlay
+
```

Comment:
This is somehow worse, since you now have a superfluous 2-way relationship between the head and a species datum, which is bad.